Packs/IdentityThreatResponsePack/Playbooks/playbook-XSOAR_-_Silverfort_for_Incident_Response.yml

```diff
@@ -0,0 +1,1037 @@
+    timertriggers: []
+    type: title
+    view: |-
+      {
+        "position": {
+          "x": 260,
+          "y": 45
+        }
+      }
+  "41":
+    form:
+      description: ""
+      expired: false
+      questions:
+      - defaultrows: []
+        fieldassociated: ""
+        gridcolumns: []
+        id: "0"
+        label: ""
+        labelarg:
+          simple: Ask manager
+        options: []
+        optionsarg: []
+        placeholder: ""
+        readonly: false
+        required: false
+        tooltip: ""
+        type: longText
+      sender: ""
+      title: Ask manager
+      totalanswers: 0
+    id: "41"
+    ignoreworker: false
+    message:
+      bcc:
+      body:
+      cc:
+      format: ""
+      methods:
+      - email
+      replyOptions:
+      - "Yes"
+      - "No"
+      subject:
+        simple: Ask manager
+      timings:
+        completeafterreplies: 1
+        retriescount: 2
+        retriesinterval: 360
+      to:
+        simple: Administrator
+    nexttasks:
+      '#none#':
+      - "42"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 0105954b-1adf-4266-88d2-740f2ea1ac6d
+      iscommand: false
+      name: Ask manager
+      type: collection
+      version: -1
+    taskid: 0105954b-1adf-4266-88d2-740f2ea1ac6d
+    timertriggers: []
+    type: collection
+    view: |-
+      {
+        "position": {
+          "x": 620,
+          "y": 895
+        }
+      }
+  "42":
+    id: "42"
+    ignoreworker: false
+    nexttasks:
+      "No":
+      - "44"
+      "Yes":
+      - "43"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: cc61427a-3d30-47dd-88ed-23e8ec5960fe
+      iscommand: false
+      name: Check response
+      type: condition
+      version: -1
+    taskid: cc61427a-3d30-47dd-88ed-23e8ec5960fe
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 620,
+          "y": 1070
+        }
+      }
+  "43":
+    id: "43"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "9"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 62589ec8-04dc-456a-8d2d-fe752d3bc03d
+      iscommand: false
+      name: Remove user from watchlist allow authentications
+      type: regular
+      version: -1
+    taskid: 62589ec8-04dc-456a-8d2d-fe752d3bc03d
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 620,
+          "y": 1340
+        }
+      }
+  "44":
+    id: "44"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "45"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      domain: {}
+      email:
+        simple: ron@ad.acaws.silverfort.io
+      sam_account: {}
+      upn:
+        simple: ron@ad.acaws.silverfort.io
```

Comment:
Please make sure that you have no problem with exposing this email address (your code is going to be part of a public repository). Please double-check all your files that include email addresses as well.